index.bs

```diff
@@ -4264,10 +4268,13 @@ calling {{CredentialsContainer/create()|navigator.credentials.create()}} they se
     Note: This concept typically leads to multiple attestation certificates. The attestation certificate requested most recently
         is called "active".
 
-    Note: [=Attestation statements=] conveying [=attestations=] of [=attestation type|type=] [=AttCA=] use the same data structure
-    as [=attestation statements=] conveying [=attestations=] of [=attestation type|type=] [=Basic=], so the two attestation types
-    are, in general, distinguishable only with externally provided knowledge regarding the contents of the [=attestation
-    certificates=] conveyed in the [=attestation statement=].
+: <dfn>Anonymization CA</dfn> (<dfn>AnonCA</dfn>)
+:: In this case, the [=authenticator=] works with a cloud-operated [=Anonymization CA=] owned by its manufacturer to dynamically generate per-[=credential=] [=attestation certificates=] on the CA such that no identification information of an individual [=authenticator=] will be revealed to [=[RPS]=] in the [=attestation statement=].
```

Comment:
Does it needs to be owned by the manufacturer?

Reply:
Perhaps owned or operated on behalf of or in cooperation with the manufacturer may cover more cases.
In the two cases I can think of as I consider Android saftyNet to be a very similar thing. They are both operated by the OS/platform manufacturer, and not neccicarily atleast in the android case the device manufacturer.

Reply:
Another question: Why does this needs to be cloud operated?